### PR TITLE
Release Notes 2.7.11

### PR DESCRIPTION
**Pull request by peytonrunyan**
[Render view](https://github.com/PrefectHQ/prefect/blob/release-notes-2.7.11/RELEASE-NOTES.md)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+- Add CLI support for work pools — https://github.com/PrefectHQ/prefect/pull/8259
+- Add fields to `work_queue` table to accommodate work pools — https://github.com/PrefectHQ/prefect/pull/8264
+
+### Documentation
+- Add git commands to Prefect Recipes contribution page — https://github.com/PrefectHQ/prefect/pull/8283
+- Add `retry_delay_seconds` and `exponential_backoff` examples to Tasks retries documentation — https://github.com/PrefectHQ/prefect/pull/8280
+- Add role permissions regarding block secrets — https://github.com/PrefectHQ/prefect/pull/8309
+- Add getting started tutorial video to Prefect Cloud Quickstart — https://github.com/PrefectHQ/prefect/pull/8336
+- Add tips for re-registering blocks from Prefect Collections — https://github.com/PrefectHQ/prefect/pull/8333
+- Add examples and corrections for Kubernetes infrastructure overrides — https://github.com/PrefectHQ/prefect/pull/8312
```
Comment (serinamarie) [inline RELEASE-NOTES.md:35]:
```suggestion
- Improve examples for Kubernetes infrastructure overrides — https://github.com/PrefectHQ/prefect/pull/8312
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Exciting New Features 🎉
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
```
Comment (serinamarie) [inline RELEASE-NOTES.md:6]:
```suggestion
- Add refactor of results into a standalone `artifact` table — https://github.com/PrefectHQ/prefect/pull/8164
```
Maybe this should go in experimental?

Comment (madkinsz) [reply RELEASE-NOTES.md:6]: Agreed this should go in experimental.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
+
+### Enhancements
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
```
Comment (madkinsz) [inline RELEASE-NOTES.md:12]: "Add functionality" is weird here, perhaps?

- Update Docker images to update preinstalled packages on build

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,50 @@
...
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
```
Comment (madkinsz) [inline RELEASE-NOTES.md:13]: I think this is more like "Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` to allow loggers to be used outside of flows" — could also elevate this into the top-level with an example.

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,50 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Exciting New Features 🎉
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
```
Comment (madkinsz) [inline RELEASE-NOTES.md:7]: @chrisguidry how should we announce this? Does this make sense to place in experimental for now or do we want to make a splash or... ?

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,50 @@
...
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
+- Fix UI to use `recentFlowRunFilter` instead of `flowRunFilter` — https://github.com/PrefectHQ/prefect/pull/8380
+
+### Experimental
+- Add ability to exclude experimental fields in API calls — https://github.com/PrefectHQ/prefect/pull/8274
+- Add work queue data migration — https://github.com/PrefectHQ/prefect/pull/8327
+- Add clean up migration — https://github.com/PrefectHQ/prefect/pull/8331
```
Comment (madkinsz) [inline RELEASE-NOTES.md:24]: These can probably be collapsed? You can link multiple pull requests for a single item

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,50 @@
...
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
+- Fix UI to use `recentFlowRunFilter` instead of `flowRunFilter` — https://github.com/PrefectHQ/prefect/pull/8380
```
Comment (madkinsz) [inline RELEASE-NOTES.md:19]: What does this mean if I'm a user?

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,50 @@
...
+### Enhancements
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
```
Comment (madkinsz) [inline RELEASE-NOTES.md:18]: What does this mean if I'm a user? This adds a default value for priority for compatibility with previous client versions, right?

File: docs/overrides/main.html [hunk 9]
```diff
@@ -2,7 +2,7 @@
 {% block announce %}
 <div style="color: white;">
   <a href="https://github.com/PrefectHQ/prefect/blob/main/RELEASE-NOTES.md#release-2710" style="color: #07e798;">
-    <strong>Prefect 2.7.10 is live!</strong>
+    <strong>Prefect 2.7.11 is live!</strong>
```
Comment (madkinsz) [inline docs/overrides/main.html:5]: Probably should add some new highlights too :)

File: RELEASE-NOTES.md [hunk 10]
```diff
@@ -1,5 +1,56 @@
 # Prefect Release Notes
 
+## Release 2.7.11
+
+### Work pools are here!
+
+We're excited to announce the release of work pools, a new feature that allows you to create logical groups for your work queues. Work pools are a great way to organize your work queues and ensure that your most important work is always prioritized. You can read more about work pools in our [documentation](https://docs.prefect.io/concepts/work-pools/).
```
Comment (billpalombi) [inline RELEASE-NOTES.md:7]:
```suggestion
We're excited to announce the release of work pools, groups of work queues. Together, work pools & queues give you greater flexibility and control in organizing and prioritizing work. You can read more about work pools in our [documentation](https://docs.prefect.io/concepts/work-pools/).
```

File: RELEASE-NOTES.md (hunk 5 above)
Comment (chrisguidry) [reply RELEASE-NOTES.md]: Yes, at most, let's say "Experimental".  We don't have it hooked up to the main Client yet.  I'd say we can either tag it experimental or quietly drop/de-emphasize it.  @WillRaphaelson what do you think?

File: RELEASE-NOTES.md [hunk 11]
```diff
@@ -1,5 +1,56 @@
...
+
+### Enhancements
+- Add refactor of results into a standalone artifact table — https://github.com/PrefectHQ/prefect/pull/8164
```
Comment (billpalombi) [inline RELEASE-NOTES.md:10]:
```suggestion
```

Comment (billpalombi) [reply RELEASE-NOTES.md:10]: I think we should remove this since its not user facing.

File: RELEASE-NOTES.md (hunk 5 above)
Comment (WillRaphaelson) [reply RELEASE-NOTES.md]: Sorry thanks for catching this guidry, yes experimental please!

File: RELEASE-NOTES.md [hunk 12]
```diff
@@ -1,5 +1,56 @@
...
+- Add Prefect Cloud Events schema and clients. — https://github.com/PrefectHQ/prefect/pull/8357
```
Comment (billpalombi) [inline RELEASE-NOTES.md:11]:
```suggestion
- Add Prefect Cloud Events schema and clients — https://github.com/PrefectHQ/prefect/pull/8357
```

File: RELEASE-NOTES.md [hunk 13]
```diff
@@ -1,5 +1,56 @@
...
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
```
Comment (billpalombi) [inline RELEASE-NOTES.md:13]:
```suggestion
- Update default task run name to exclude the task key hash — https://github.com/PrefectHQ/prefect/pull/8292
```

File: RELEASE-NOTES.md (hunk 10 above)
Comment (billpalombi) [inline RELEASE-NOTES.md:7]:
```suggestion
```

**Review by billpalombi — COMMENTED** [review]
Some minor nits

File: RELEASE-NOTES.md [hunk 14]
```diff
@@ -1,5 +1,56 @@
...
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
+- Fix UI to use `recentFlowRunFilter` instead of `flowRunFilter` — https://github.com/PrefectHQ/prefect/pull/8380
+
+### Experimental
+- Add ability to exclude experimental fields in API calls — https://github.com/PrefectHQ/prefect/pull/8274
```
Comment (billpalombi) [inline RELEASE-NOTES.md:24]:
```suggestion
- We're working on work pools, groups of work queues. Together, work pools & queues give you greater flexibility and control in organizing and prioritizing work.
- Add ability to exclude experimental fields in API calls — https://github.com/PrefectHQ/prefect/pull/8274
```

**Review by billpalombi — COMMENTED** [review]
Moving work pools to experimental

File: RELEASE-NOTES.md (hunk 13 above)
Comment (madkinsz) [reply RELEASE-NOTES.md:12]: Well... it's a hash of the "task key" there is no "task key hash" but this reads nicely too.

Comment (billpalombi) [reply RELEASE-NOTES.md:8]: haha fair I'll drop it

File: RELEASE-NOTES.md [hunk 15]
```diff
@@ -1,5 +1,51 @@
...
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
+- Fix Kubernetes job watch exit when no timeout given — https://github.com/PrefectHQ/prefect/pull/8350
+- Fix default value for priority on `WorkQueue` core schema — https://github.com/PrefectHQ/prefect/pull/8373
+- Fix UI to use `recentFlowRunFilter` instead of `flowRunFilter` — https://github.com/PrefectHQ/prefect/pull/8380
+
+### Experimental
+- We're working on work pools, groups of work queues. Together, work pools & queues give you greater flexibility and control in organizing and prioritizing work.
```
Comment (billpalombi) [inline RELEASE-NOTES.md:19]:
```suggestion
- We're working on work pools, groups of work queues. Together, work pools & queues give you greater flexibility and control to organize and prioritize work.
```

File: RELEASE-NOTES.md [hunk 16]
```diff
@@ -1,5 +1,51 @@
...
+## Release 2.7.11
+
+### Enhancements
+- Update pinned pip version in Docker images — https://github.com/PrefectHQ/prefect/pull/8289
+- Update default task run name to exclude hash of task key — https://github.com/PrefectHQ/prefect/pull/8292
+- Add functionality that updates preinstalled packages in Docker images — https://github.com/PrefectHQ/prefect/pull/8288
+- Add `PREFECT_LOGGING_ORION_WHEN_MISSING_FLOW` setting to handle error raised when logger called outside of flow run context — https://github.com/PrefectHQ/prefect/pull/8311
+
+### Fixes
+- Fix cancellation bug — https://github.com/PrefectHQ/prefect/pull/8315
```
Comment (billpalombi) [inline RELEASE-NOTES.md:12]:
```suggestion
- Fix bug preventing agents from properly updating Cancelling runs to a Cancelled state — https://github.com/PrefectHQ/prefect/pull/8315
```

File: RELEASE-NOTES.md (hunk 8 above)
Comment (billpalombi) [reply RELEASE-NOTES.md:14]: We should move this to experimental

File: RELEASE-NOTES.md (hunk 7 above)
Comment (billpalombi) [reply RELEASE-NOTES.md:14]: We can just remove this. It was part of work pool development

**Review by billpalombi — COMMENTED** [review]
Adding PRs

**Review by billpalombi — APPROVED** [review]
lgtm

**Review by billpalombi — APPROVED** [review]
Looks great!

**Review by madkinsz — APPROVED** [review]

**Pull request merged**